packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/SubjectReductions/SubjectReductions.js

```diff
@@ -0,0 +1,33 @@
+
+import axios from 'axios'
+import { PropTypes } from 'mobx-react';
+
+const REDUCER_KEY = 'machine-learnt';
+
+function SubjectReductions({ workflowId, subjectId, cb }) {
+  const query = `{
+    workflow(id: ${workflowId}) {
+      subject_reductions(subjectId: ${subjectId}, reducerKey:"${REDUCER_KEY}")
+      {
+        data
+      }
+    }
+  }`
+
+  axios.post('https://caesar-staging.zooniverse.org/graphql', {
```

Comment:
We have the GraphQL client installed for Caesar requests, so prefer that instead of axios.

You can access it via `classifierStore.client.caesar`.
https://github.com/zooniverse/front-end-monorepo/blob/103fc15dd0428a843d7c1a85a93b4951c9edb67e/packages/lib-classifier/src/components/Classifier/ClassifierContainer.js#L30-L35

Take a look at the transcription task for an example of a drawing tool that loads in drawn marks from caesar, then renders them.
https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/story/drawing-tools-transcribedlines--default&globals=locale:en;locales.en:English;locales.test:Test+Language

Reply:
Here’s the code that fetches Caesar reductions for transcription subjects. 
https://github.com/zooniverse/front-end-monorepo/blob/90d72c7120620e7ce7b892ce92c579fc103d906a/packages/lib-classifier/src/store/subjects/Subject/TranscriptionReductions/TranscriptionReductions.js#L90-L114

Reply:
In #4237, I've started work on a hook that fetches reductions for transcription tasks. That might be useful here too.

Reply:
As discussed with the team I will be removing this code in favor of the hook in a separate PR.